README.md: add mac OS mock service worker instructions (#1436)
RedHatInsights/image-builder-frontend: README.md

4. open browser at `https://stage.foo.redhat.com:1337/beta/insights/image-builder`

> if you want to develop in environment that automatically generated data, run the command `npm run stage-beta:msw`.

Reviewer: Is the data generated on the fly or mocked?

Also if it's generated
> environment _with_ generated data

4. open browser at `https://stage.foo.redhat.com:1337/beta/insights/image-builder`

> 

Reviewer: stray newline

```bash
npm run api
```

## Mac Configurations

> if you want to develop in environment that automatically generated data, run the command npm run stage-beta:msw.

Reviewer: ```suggestion
### Using MSW data
If you want to develop in environment with automatically generated data, run the command `npm run stage-beta:msw`.
```

To enable msw, you must configure SSL certification on your MAC computer.

Follow these steps to find and paste the certification file into the 'Keychain Access' application:

Open the 'Keychain Access' application.

Select 'login' on the left side.

Navigate to the 'Certificates' tab.

> Drag the certification file (usually located at /image-builder-project/image-builder-frontend/node_modules/.cache/webpack-dev-server/server.pem) to the certification list.

Reviewer 2: The path can start with the frontend folder.

```suggestion
Drag the certification file (usually located at /image-builder-frontend/node_modules/.cache/webpack-dev-server/server.pem) to the certification list.
```

Double-click on the added certificate (localhost certificate) to open the localhost window.

Open the 'Trust' dropdown menu.

Set all options to 'Always Trust'.

> Close the localhost screen.

Reviewer: Can we make this a numbered list?

```
1. Open the 'Keychain Access' application.
2. ...
```

> Run npm run stage-beta:msw and open the Firefox browser to verify that it is working as expected.

Reviewer 2: ```suggestion
Run `npm run stage-beta:msw` and open the Firefox browser to verify that it is working as expected.
```

```bash
npm run api
```

## Mac Configurations

> if you want to develop in environment that automatically generated data, run the command npm run stage-beta:msw. To enable msw, you must configure SSL certification on your computer.

Reviewer: ```suggestion
### Using MSW data
If you want to develop in environment with automatically generated data, run the command `npm run stage-beta:msw`. To enable msw, you must configure SSL certification on your computer.
```
